wikipedia-dark.user.css

```diff
@@ -1276,9 +1276,11 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
   .messagebox, .errorbox, .warningbox, .successbox,
   .suggestions-special .special-label, .reference-text,
   .mwe-popups .mwe-popups-container, #viewsourcetext {
-    background-color: var(--gray-2) !important;
```

Comment:
Why is this being removed for `.messagebox, .errorbox, ...`? It would be better to remove the `.mwe-popups .mwe-popups-container` selector from this group than to break the group.

Reply:
The code was like that before my previous changes, so this was like a revert to the previous code. Apparently, we still need the color style for this tag

Reply:
You may take a look at my previous commit here: https://github.com/StylishThemes/Wikipedia-Dark/commit/8cd3388d0c51419c91f5c3fee923ceed2f1a041e. So in this commit, I reverted the change and apply it on a separate selector below because I wasn't sure how it would behave on the other selectors in the group.